.github/workflows/build-deploy.yml

```
@@ -145,7 +145,7 @@ jobs:
           file: ./docker/prod/Dockerfile
           build-args: |
             SENTRY_RELEASE="${{ needs.configure.outputs.IMAGE_TAG }}"
-            GIT_SHA="${{ github.sha }}}"
+            GIT_SHA=${{ github.sha }}}
```

just curious, why this expression should not be quoted? Also there is extra closed `}`
```suggestion
            GIT_SHA=${{ github.sha }}
```

---

oh! maybe that extra `}` is the problem! thanks for noticing, I am adding back quotation marks and fixing this.